### DISCUSSION TITLE
Skinning - Allow elements in front of the UI

### DISCUSSION THREAD

**Discussion by PuddlePumpkin** [ppy/osu#23991] [Ideas]
Would be useful for skinners to be able to use "bring to front" to move skin elements in front of things like the options at the bottom of song select as well as the logo 
![image_2023-06-21_124150608](https://github.com/ppy/osu/assets/66428305/0e8aed71-5fa5-4460-a796-ffb38827a9f9)

**Comment by peppy** [comment]
This is planned eventually – we will provide multiple layer targets to put things at the depth you want them.
